pkg/monitor/monitor/monitor_manager.go

```diff
@@ -82,7 +86,26 @@ func NewMonitorManager(
 func (mm *MonitorManager) SyncMonitor(monitor *v1alpha1.TidbMonitor) error {
 
 	if monitor.DeletionTimestamp != nil {
-		return nil
+		return mm.patchTidbClusterStatus(nil, monitor)
```

Comment:
Is this necessary since you have added the status sync in tidblcuster?

Reply:
removed